fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ReviewQualityCheckScaffold.kt

```diff
@@ -27,31 +28,36 @@ import androidx.compose.ui.semantics.semantics
 import androidx.compose.ui.unit.dp
 import org.mozilla.fenix.R
 import org.mozilla.fenix.compose.BottomSheetHandle
-import org.mozilla.fenix.compose.annotation.LightDarkPreview
-import org.mozilla.fenix.shopping.state.ReviewQualityCheckState
 import org.mozilla.fenix.theme.FirefoxTheme
 
 private val bottomSheetShape = RoundedCornerShape(topStart = 16.dp, topEnd = 16.dp)
 private const val BOTTOM_SHEET_HANDLE_WIDTH_PERCENT = 0.1f
 
 /**
- * Top-level UI for the Review Quality Check feature.
+ * A scaffold for review quality check UI that implements the basic layout structure with
+ * [BottomSheetHandle], [Header] and [content].
  *
- * @param onRequestDismiss Invoked when a user actions requests dismissal of the bottom sheet.
+ * @param onRequestDismiss Invoked when a user action requests dismissal of the bottom sheet.
  * @param modifier The modifier to be applied to the Composable.
+ * @param content The content of the bottom sheet.
  */
 @Composable
-fun ReviewQualityCheckContent(
+fun ReviewQualityCheckScaffold(
```

Comment:
nit: feels like if we use `scaffold` here, it sets a new naming precedent? How about just `ReviewQualityCheck`?

Reply:
It is actually a scaffold similar to the [Scaffold](https://developer.android.com/reference/kotlin/androidx/compose/material/package-summary#scaffold) as in there is a slot for the content along with the basic layout structure. I think `ReviewQualityCheck` doesn't say more but less 🤔  don't you think?

Reply:
Sure, I think the `Scaffold` naming itself is fine, it was more a comment for us to have consistency with our naming of Composables. Agree `ReviewQualityCheck`doesn't say more, but (I am maybe wrong here) it'd be assumed this was the root/parent of the feature. If this is a new type/style of Composable then establishing this as a convention here is okay, but let's try to communicate that in some way 👍

Reply:
Yes I agree, I think we as a team need to create and settle on a style guide for things like this which can't be linted, this is something we should talk about 👍